go/logic/server.go

```diff
@@ -163,8 +163,7 @@ no-throttle                          # End forced throttling (other throttling m
 unpostpone                           # Bail out a cut-over postpone; proceed to cut-over
 panic                                # panic and quit without cleanup
 help                                 # This message
-- use '?' (question mark) as argument to get info rather than set. e.g. "max-load=?" will just print out current max-load.
-`)
+- use '?' (question mark) as argument to get info rather than set. e.g. "max-load=?" will just print out current max-load.\n\n`)
```

Comment:
I first just dropped the newline, but then I realized it would change the output here. So I went the `Fprintf()` route AND had to escape the `70% r...` part of the string